app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -17,43 +16,15 @@ import edu.uc.hickmadc.customnotifications.ui.main.dialogfragment
 
 class MainActivity : AppCompatActivity() {
 
-
-
-
-
-
-    //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
-    val description = arrayOf<String>(
-        "Brush teeth for 2 minutes",
-        "Eat so you don't feel like shit",
-        "")
-    val days= arrayOf<String>("MWF","TFS","WTF")
-    val times= arrayOf<String>("1:30","13:15","8:30")
-
     override fun onCreate(savedInstanceState: Bundle?) {
 
-
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
 
-
-        val lView= findViewById<ListView>(R.id.listView)
-        val myListAdapter = MyListAdapter(this,language,description,days, times)
-        lView.adapter = myListAdapter
         if (savedInstanceState == null) {
             setupBottomNavigationBar()
         }
-        val addButton = findViewById<ImageButton>(R.id.btnAdd)
-        addButton.setOnClickListener{
-        var dialog= dialogfragment()
-
-        dialog.show(supportFragmentManager, "customdialog")
-
-
-        }
-
     }
```

Comment:
This code got moved over to the main fragment. You can read my justification for that in some of the other comments below. In short though, I moved your notification logic over to your main notification fragment, the main fragment class.

Reply:
Good.  DRY principle.